src/main/java/org/jitsi/jigasi/JvbConference.java

```diff
@@ -2121,4 +2136,52 @@ public void processStanza(Stanza stanza)
             }
         }
     }
+
+    /**
+     * Used to check the jvb side of the call for any activity.
+     */
+    private class MediaActivityChecker
+        extends TimerTask
+    {
+        @Override
+        public void run()
+        {
+            // if the call was stopped before we check ignore
+            if (!started)
+            {
+                return;
+            }
+
+            CallPeer peer = jvbCall.getCallPeers().next();
+            if (peer instanceof MediaAwareCallPeer)
+            {
+                MediaAwareCallPeer peerMedia = (MediaAwareCallPeer) peer;
+
+                CallPeerMediaHandler mediaHandler = peerMedia.getMediaHandler();
+                if (mediaHandler != null)
+                {
+                    MediaStream stream = mediaHandler.getStream(MediaType.AUDIO);
+                    if (stream != null && stream instanceof AudioMediaStreamImpl)
+                    {
+                        try
+                        {
+                            // if there is no activity on the audio channel this means there is a problem
+                            // establishing the media path with the bridge so we can just fail the call
+                            if (((AudioMediaStreamImpl) stream).getLastInputActivityTime() == 0)
```

Comment:
Looking at the impl, it returns `-1` when there are no input streams. Do you want to include `-1` here, or are there special semantics for `0` that I didn't find?

Reply:
Ok, I can change it to `<= 0`. The 0 came from my testing ... I changed the code in a way that DTLS was not established and this was returning 0 ...